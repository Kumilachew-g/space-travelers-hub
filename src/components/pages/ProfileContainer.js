// import ProfileMissions from '../missions/ProfileMissions';
import ProfileRocket from '../rockets/ProfileRocket';

const ProfileContainer = () => (
  <div className="profileContainer">
    <ProfileRocket />
    {/* <ProfileMissions /> */}
  </div>
);

export default ProfileContainer;
