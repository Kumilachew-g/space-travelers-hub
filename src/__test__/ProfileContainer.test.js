import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import ProfileContainer from '../components/pages/ProfileContainer';
import store from '../redux/cofigureStore';

describe('Component test snapshot', () => {
  it('Profile renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <ProfileContainer />
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
