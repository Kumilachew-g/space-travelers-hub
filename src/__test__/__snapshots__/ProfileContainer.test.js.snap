// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component test snapshot Profile renders correctly 1`] = `
<div
  className="d-flex justify-content-between profile-list "
>
  <div
    className="profile-rocket-container"
  >
    <h2>
      My Rockets
    </h2>
    <ul
      className="profile-rocket-list-conatiner"
    >
      <div
        className="status"
      >
        No Rockets reserved
      </div>
    </ul>
  </div>
  <section>
    <li
      className="list-group-item"
    >
      <h1>
        My Missions
      </h1>
    </li>
    <ul
      className="list-group my-3"
    >
      <li
        className="list-group-item"
      >
        <span>
          No reserved missions.
        </span>
      </li>
    </ul>
  </section>
</div>
`;
